### ISSUE TITLE
Would be great for the future to add a real readme

### ISSUE THREAD

**Issue by doliG** [xmazella/ethglobal#1]
I was thinking that for the future it would be great to add a readme explaining in a few lines what this project is, what problem it aims to solve and also few screenshots :)

@xmazella 👀

**Comment by MatteoMer** [comment]
ok you can do it

**Comment by doliG** [comment]
yup, I've started and put a brief skeleton but need to dig more into what we did bc my memory is blurry haha. @xmazella any help ? 🙏
